### ISSUE TITLE
Remove dependency on AspectJ

### ISSUE THREAD

**Issue by ledsoft** [kbss-cvut/termit#318]
Now that JOPA does not require AspectJ, TermIt depends on AspectJ only because some cross-cutting behavior is implemented by aspects. We should replace this by other suitable means (e.g., Spring application events) and remove the dependency on AspectJ.

**Comment by lukaskabc** [comment]
Termit uses AspectJ for `@Configurable` annotation that allows to autowire `Configuration` bean into Entities (Vocabulary, Term).
The configuration is used there to acquire a language (`config.getPersistence().getLanguage()`).

Task throttling was also implemented as a Spring AOP aspect, so it should not require AspectJ compile time weaving.

I think https://github.com/kbss-cvut/termit-ui/issues/521 should be implemented first, which should remove autowiring configuration in entities.

**Comment by ledsoft** [comment]
I see. However, from what I saw, the injected `Configuration` bean is used only in the `get/setPrimaryLabel` method. The setter is used only in tests and the getter should be possible to remove with some refactoring. We would want to remove these methods if we implemented kbss-cvut/termit-ui#521 anyway.

I would thus propose to start with this, removing the `get/setPrimaryLabel` methods so that we can get rid of `@Configurable`, make a PR, and then continue with removing AspectJ. Do you think it is doable this way?

**Comment by lukaskabc** [comment]
Right, I should have checked where those accessors are used, it shouldn't be a problem to refactor that.
